CollAction/Frontend/app/project/FindProject.tsx

```diff
@@ -36,7 +37,16 @@ export default class FindProject extends React.Component<IFindProjectProps, IFin
 
     // Fetch projects with out filters set
     const getUrl: () => string = () => {
-      return "/api/projects/find";
+      let url = "/api/projects";
```

Comment:
When testing, we should ensure that the /find page still works. This shouldn't break it, but it does touch its code.

In the future, if these components start drifting apart in functionality, we should split them apart.